extensions/amp-story/1.0/story-analytics.js

```diff
@@ -27,6 +27,10 @@ export const StoryAnalyticsEvent = {
   STORY_CONTENT_LOADED: 'story-content-loaded',
   STORY_MUTED: 'story-audio-muted',
   STORY_UNMUTED: 'story-audio-unmuted',
+  SHOPPING_TAG: 'story-shopping-tag',
+  SHOPPING_BUY_NOW: 'story-shopping-buy-now',
+  SHOPPING_PLP_CARD: 'story-shopping-plp-card',
+  SHOPPING_PRODUCT_DETAILS: 'story-shopping-product-details',
```

Comment:
Should these describe the action? 
Such as `SHOPPING_BUY_NOW_CLICK` and `SHOPPING_PRODUCT_DETAILS_VIEW`?

Reply:
Sounds good, I'll make them more descriptive.